src/scenes/AcceleratorRouter/DeliverableView.tsx

```diff
@@ -1,17 +1,15 @@
+import React from 'react';
 import { BusySpinner, Message } from '@terraware/web-components';
-import { useRouteMatch } from 'react-router-dom';
-import { APP_PATHS } from 'src/constants';
 import strings from 'src/strings';
 import useDeviceInfo from 'src/utils/useDeviceInfo';
 import Card from 'src/components/common/Card';
 import Page from 'src/components/Page';
-import DocumentsList from 'src/scenes/DeliverablesRouter/DocumentsList';
 import AcceleratorDocumentsList from 'src/scenes/AcceleratorRouter/AcceleratorDocumentsList';
-import { EditProps, ViewProps } from './types';
-import TitleBar from './TitleBar';
-import StatusBar from './StatusBar';
-import Description from './Description';
-import DocumentsUploader from './DocumentsUploader';
+import { EditProps, ViewProps } from 'src/components/DeliverableView/types';
+import TitleBar from 'src/components/DeliverableView/TitleBar';
+import StatusBar from 'src/components/DeliverableView/StatusBar';
+import Description from 'src/components/DeliverableView/Description';
+import DocumentsUploader from 'src/components/DeliverableView/DocumentsUploader';
```

Comment:
Eventually these might be replaced with view-specific components that wrap the shared component